.github/workflows/docker-lint.yaml

```
@@ -0,0 +1,17 @@
+name: docker-lint
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'Dockerfile'
```

```suggestion
      - 'Dockerfile'
      - '.github/workflows/docker-lint.yaml'
```

So that we are sure we don't break the workflow when changing it